ios/AppLovinMAXAdView.m

```diff
@@ -23,7 +23,7 @@ - (void)setAdUnitId:(NSString *)adUnitId
     // Ad Unit ID must be set prior to creating MAAdView
     if ( self.adView )
     {
-        [[AppLovinMAX shared] log: @"Attempting to set Ad Unit ID %@ after MAAdView is created", adUnitId];
+        [AppLovinMAX.shared log: @"Attempting to set Ad Unit ID %@ after MAAdView is created", adUnitId];
```

Comment:
We can keep `[AppLovinMAX shared]` - general iOS ObjC convention for referencing shared instances

Reply:
OK, got it, we'll keep `[AppLovinMAX shared]`.